docs-client/src/lib/versions.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface Version {
+  artifactId: string;
+  artifactVersion: string;
+  commitTimeMillis: number;
+  shortCommitHash: string;
+  longCommitHash: string;
+  repositoryStatus: string;
+}
+
+export function extractSimpleArtifactVersion(artifactVersion: string): string {
+  const simpleArtifactVersion = artifactVersion.match(
+    /^[0-9]+\.[0-9]+\.[0-9]+/,
+  );
+  return simpleArtifactVersion ? simpleArtifactVersion.toString() : '';
+}
+
+export function convertLongToUTCDate(date: number): string {
+  return new Date(date).toUTCString();
+}
+
+export function formatReadableTime(datetime: number) {
```

Comment:
Perhaps we could just use day.js, so we don't have to maintain this by ourselves? https://github.com/iamkun/dayjs